webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala

```diff
@@ -175,11 +175,11 @@ final case class OntologyResponderV2Live(
       entitiesInWrongSchema =
         (standoffClassIris ++ standoffPropertyIris).filter(_.getOntologySchema.contains(ApiV2Simple))
 
-      _ = if (entitiesInWrongSchema.nonEmpty) {
-            throw NotFoundException(
-              s"Some requested standoff classes were not found: ${entitiesInWrongSchema.mkString(", ")}"
-            )
-          }
+      _ <- ZIO.fail {
+             throw NotFoundException(
```

Comment:
Do not throw the `NotFoundException`, in a `ZIO.fail` an exception becomes simply a value as any other.

```suggestion
             NotFoundException(
```